terraform-provider-kentik-cloudexport/.goreleaser.yml

```diff
@@ -0,0 +1,54 @@
+# Visit https://goreleaser.com for documentation on how to customize this
```

Comment:
Note: this file is part of terraform provider publishing process and will be updated later